packages/runtime/client/rest/src/contract/details.ts

```diff
@@ -50,6 +50,7 @@ export interface ContractDetails extends t.TypeOf<typeof ContractDetails> {}
  * This is a {@link !io-ts-usage | Dynamic type validator} for the {@link ContractDetails:type}.
  * @category Validator
  */
+// DISCUSSION : Tags are missing
```

Comment:
Do you mean that the backend is not adding the Tags information and this is a discussion if we should add them?
Or you meant to add a note to add Tags already available in the backend?

Reply:
I mean yeah, it is available in the backend and we don't have them